tools/Dockerfile

```diff
@@ -4,29 +4,29 @@ FROM ubuntu:20.04 as builder
 ARG DEBIAN_FRONTEND=noninteractive
 ARG TARGETARCH
 # https://github.com/kubernetes/kubernetes/releases
-ARG KUBECTL_VERSION=1.30.4
+ARG KUBECTL_VERSION=1.31.1
 # https://github.com/helm/helm/tags
-ARG HELM_VERSION=3.15.1
+ARG HELM_VERSION=3.16.1
 # https://github.com/databus23/helm-diff/releases
-ARG HELM_DIFF_VERSION=3.9.7
+ARG HELM_DIFF_VERSION=3.9.11
 # https://github.com/jkroepke/helm-secrets/releases
-ARG HELM_SECRETS_VERSION=3.15.0
+ARG HELM_SECRETS_VERSION=4.6.1
```

Comment:
This upgrade has breaking changes. https://github.com/jkroepke/helm-secrets/blob/main/CHANGELOG.md#400---2022-09-11
I think we can solve this by updating `src/common/crypt.ts`

```
enum CryptType {
  ENCRYPT = 'helm secrets encrypt -f',
  DECRYPT = 'helm secrets decrypt -f',
  ROTATE = 'sops --input-type=yaml --output-type=yaml -i -r',
}
```